### PR TITLE
Add orange-canvas-core recipe to conda-forge

### PR DESCRIPTION
**Pull request by ales-erjavec**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/orange-canvas-core/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    the Orange Canvas (http://orange.biolab.si) data-mining application
+    (for which it was developed in the first place).
+
+  doc_url: https://orange-canvas-core.readthedocs.io/
+  dev_url: https://github.com/biolab/orange-canvas-core
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
```
Comment (marcelotrevisani) [inline recipes/orange-canvas-core/meta.yaml]: Could you please remove those comments?

File: recipes/orange-canvas-core/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    - ales-erjavec
+    - lanzagar
+    - markotoplak
+    - primozgodec
```
Comment (marcelotrevisani) [inline recipes/orange-canvas-core/meta.yaml]: @lanzagar @markotoplak @primozgodec
Could you please confirm that you want to be a maintainer of this recipe as well?

File: recipes/orange-canvas-core/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install -vv --no-deps . "
```
Comment (marcelotrevisani) [inline recipes/orange-canvas-core/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/orange-canvas-core/meta.yaml (hunk 2 above)
Comment (PrimozGodec) [reply recipes/orange-canvas-core/meta.yaml]: I am confirming at I am OK to be a maintainer

Comment (markotoplak) [reply recipes/orange-canvas-core/meta.yaml]: I am confirming that I would like to be a maintainer.

Comment (lanzagar) [reply recipes/orange-canvas-core/meta.yaml]: I also confirm I would like to be a maintainer.

**Comment by ales-erjavec** [comment]
ping @conda-forge/staged-recipes

**Pull request merged**